projects/rocr-runtime/runtime/hsa-runtime/core/runtime/runtime.cpp

```diff
@@ -1096,7 +1096,13 @@ hsa_status_t Runtime::PtrInfo(const void* ptr, hsa_amd_pointer_info_t* info, voi
   // Ex. ROCR_VISIBLE_DEVICES or peer GPU is not supported by ROCm.
   retInfo.agentOwner.handle = 0;
   auto nodeAgents = agents_by_node_.find(thunkInfo.Node);
-  assert(nodeAgents != agents_by_node_.end() && "Node id not found!");
+  if (nodeAgents == agents_by_node_.end()) {
```

Comment:
If the pointer_info API is called after hsa_shut_down(), I think we should fail at the very beginning of the API call.

Is the IS_OPEN() macro not catching this?